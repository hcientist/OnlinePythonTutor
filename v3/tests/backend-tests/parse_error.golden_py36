{
  "code": "x = 0\nfor i in range(10):\n  x += 1\n   print x\n  x += 1\n\n",
  "trace": [
    {
      "event": "uncaught_exception",
      "line": 4,
      "offset": 3,
      "exception_msg": "IndentationError: unexpected indent (<string>, line 4)"
    }
  ]
}
