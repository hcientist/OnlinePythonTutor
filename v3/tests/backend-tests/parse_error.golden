{
  "code": "x = 0\nfor i in range(10):\n  x += 1\n   print x\n  x += 1\n\n", 
  "trace": [
    {
      "exception_msg": "IndentationError: unexpected indent (<string>, line 4)", 
      "line": 4, 
      "event": "uncaught_exception", 
      "offset": 3
    }
  ]
}
